src/DankCord/Objects.py

```diff
@@ -183,16 +183,32 @@ def __init__(self, data: dict) -> None:
         self.dropdowns: list = [
             item for component in self.components for item in component.components if isinstance(item, Dropdown)
         ]
+        try:
+            self.nonce: str = data["nonce"]
+        except:
+            # Unless used wrongly, it just means the object was used on a MESSAGE_UPDATE data, those don't have nonce
+            self.nonce: str = ""
 
 
 class Bot:
     """
     Represents the bot account.
     """
-
+    
     def __init__(self, data: dict) -> None:
         self.username: str = data["d"]["user"]["username"]
         self.id: int = int(data["d"]["user"]["id"])
         self.discriminator: int = int(data["d"]["user"]["discriminator"])
         self.email: str = data["d"]["user"]["email"]
         self.bot: str = f"{self.username}#{self.discriminator}"
+
+class CommandResult:
+    """
+    Represents a class that has the result of running a certain command.
+    """
+    
+    def __init__(self, success: bool, death: bool, gain: dict, loss: dict) -> None:
```

Comment:
Could make `DankItem` object and typehint it as that here, The object will need to be created later for market scraping and inventory scraping anyways, should be better this way.

Reply:
The class isnt being used in the current code at this point anyway, so I'll probably do that in the next PR or so.